source/OptimaJet.DWKit.StarterApplication/Repositories/ProjectRepository.cs

```diff
@@ -37,6 +38,32 @@ public override IQueryable<Project> Filter(IQueryable<Project> query, FilterQuer
                 return query.FilterByOrganization(filterQuery, allowedOrganizationIds: orgIds);
             }
 
+            var value = filterQuery.Value;
+            var op = filterQuery.Operation.ToEnum<FilterOperations>(defaultValue: FilterOperations.eq);
+
+            if (filterQuery.Has(PROJECT_PRODUCT_UPDATED_DATE)) {
```

Comment:
JADNC doesn't yet have a build in way to expose querying on relationships like this, so these custom query keys need to be interpreted in the repository